File: src/models/Media.js
export class Media {
  constructor(id, type, userId, title, caption, url, tags) {
    this.id = id; // int
    this.type = type; // int
    this.userId = userId; // int
    this.title = title; // string
    this.caption = caption; // string
    this.url = url; // string
    this.tags = tags; // string[]
  }
}

File: Backend/WorthyWebAppChallengeBackend/Models/Media.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WorthyWebAppChallengeBackend.Models
{
    public class Media
    {
        public int Id { get; set; }

        public int Type { get; set; }

        public int UserId { get; set; }

        public string Title { get; set; }

        public string Caption { get; set; }

        public string Url { get; set; }

        public IList<string> Tags { get; set; }
    }
}
